frontend/public/features.ts

```diff
@@ -45,6 +46,7 @@ export enum FLAGS {
   CAN_LIST_CRD = 'CAN_LIST_CRD',
   CAN_CREATE_PROJECT = 'CAN_CREATE_PROJECT',
   PROJECTS_AVAILABLE = 'PROJECTS_AVAILABLE',
+  SERVICE_CATALOG_AVAILABLE = 'SERVICE_CATALOG_AVAILABLE',
```

Comment:
I'd just call this `SERVICE_CATALOG`. Most flags use the feature name without the available suffix

Reply:
Yeah, one of you will need to rebase depending on what merges first :/